.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: node_js
+
+cache:
+  directories:
+    - '$HOME/.npm'
+
+notifications:
+  email: false
+
+node_js:
+  - '10'
+  - '8'
+
+install: npm ci
+script:
+  - npm run test
```

Comment:
If `npm run test:prod` is part of 'Coverage' job then is it required here?

Reply:
here it will run tests on the versions specified here:
```
node_js:
  - '10'
  - '8'
```